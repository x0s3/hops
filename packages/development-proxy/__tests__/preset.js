/* eslint-env jest */

describe('development proxy preset', () => {
  it('should match snapshot', () => {
    expect(require('../preset')).toMatchSnapshot();
  });
});
