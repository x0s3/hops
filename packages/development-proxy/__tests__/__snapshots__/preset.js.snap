// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`development proxy preset should match snapshot 1`] = `
Object {
  "mixins": Array [
    "/home/zaubernerd/workspace/xing/hops/packages/development-proxy",
  ],
}
`;
